src/layers/legacy/medCoreLegacy/parameters/medDoubleParameterL.cpp

```diff
@@ -91,16 +91,16 @@ void medDoubleParameterL::setValue(double value)
         }
 
         //  update intern widget
-        this->blockInternWidgetsSignals(true);
```

Comment:
Cela change quoi de les avoir déplacer dans l'update du widget?

Reply:
this allows to put back the `updateInternWigets` line in `setRange` function which have been removed in a previous PR, but keeping the blocks. If we use `updateInternWigets` we need to block

Reply:
ok